pkg/steps/bundle_source.go

```diff
@@ -69,58 +69,49 @@ func (s *bundleSourceStep) run(ctx context.Context) error {
 	return handleBuild(ctx, s.buildClient, build, s.artifactDir)
 }
 
-func replaceCommand(manifestDir, pullSpec, with string) string {
-	return fmt.Sprintf("find %s -type f -exec sed -i 's?%s?%s?g' {} +", manifestDir, pullSpec, with)
+func replaceCommand(pullSpec, with string) string {
+	return fmt.Sprintf("find . -type f -name \\*.yaml -exec sed -i 's?%s?%s?g' {} +", pullSpec, with)
```

Comment:
Must these be YAML files?

Reply:
AFAIK yes, they are always manifest files. Technically they could be called differently, I guess. `.yml` or something like that.